src/server/middleware/twitter.middleware.ts

```diff
@@ -304,6 +313,172 @@ function loadTwitterData(
     });
 }
 
+function loadTwitterDataV2(
+  credentials: TwitterApiCredentials,
+  request: GetTweetsRequest
+): Promise<TwitterApiResponse> {
+  const requestUrl = 'https://api.twitter.com/2/tweets/search/all';
+  const user = request.credentials?.additionalUserInfo?.username;
+  if (!user) {
+    throw new Error('No user credentials in GetTweetsRequest');
+  }
+
+  const params = {
+    // Include next_token if it's part of the request.
+    ...(request.nextPageToken && { next_token: request.nextPageToken }),
+    ...{
+      query: `(@${user} OR url:twitter.com/${user}) -from:${user} -is:retweet`,
+      max_results: BATCH_SIZE,
+      'user.fields': 'id,name,username,profile_image_url,verified',
```

Comment:
Mostly just curious: I haven't seen this syntax before; is it a new Typescript feature?